provider/spark_test.go

```diff
@@ -99,7 +111,10 @@ func testRegisterResource(store *SparkOfflineStore) error {
 	if !reflect.DeepEqual(fetchedTable, table) {
 		return fmt.Errorf("Did not properly register table")
 	}
-	return nil
+
```

Comment:
Should make this function an actual Go test

Reply:
it's functional rn to inherit the initialization